client/verta/tests/test_experimentrun/test_deployment.py

```diff
@@ -157,13 +157,13 @@ def test_no_model_deploy_error(self, experiment_run, model_for_deployment):
             "DELETE",
             "{}://{}/api/v1/modeldb/experiment-run/deleteArtifact".format(experiment_run._conn.scheme,
                                                               experiment_run._conn.socket),
-            experiment_run._conn, json={'id': experiment_run.id, 'key': _artifact_utils.MODEL_KEY}
+            experiment_run._conn, json={'id': experiment_run.id, 'key': experiment_run._MODEL_KEY}
         )
         _utils.raise_for_http_error(response)
 
         with pytest.raises(RuntimeError) as excinfo:
             experiment_run.deploy()
-        assert _artifact_utils.MODEL_KEY in str(excinfo.value)
+        assert experiment_run._MODEL_KEY in str(excinfo.value)
```

Comment:
These tests aren't actually being executed anymore, but I've made the change here for completeness.